chapter06/drjt/ex1.pl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/perl
+use 5.010;
+use warnings;
+use strict;
+use Storable qw( nstore retrieve );
+
+my $storage = 'coconet.archive';
+
+my %total_bytes;
+if ( -r $storage ) {
+  %total_bytes = %{ retrieve( $storage ) };
+}
```

Comment:
I don't think there's any need to make `total_bytes` a hash rather than a hashref. If it was a hashref you wouldn't have to de-reference the output of `retrieve` and re-reference it again for `nstore`.